src/cli/cli.cpp

```diff
@@ -0,0 +1,123 @@
+#include "cli.hpp"
+#include "states.hpp"
+#include "Particle.h"
+#include "conio.hpp"
+#include "menuItems/systemCommands.hpp"
+
+#include <fstream>
```

Comment:
In general, there is a specific order of includes.

1. The prototype/interface header for this implementation (ie, the .h/.hh file that corresponds to this .cpp/.cc file).
2. Other headers from the same project, as needed.
3. Headers from other non-standard, non-system libraries (for example, Qt, Eigen, etc).
4. Headers from other "almost-standard" libraries (for example, Boost)
5. Standard C++ headers (for example, iostream, functional, etc.)
6. Standard C headers (for example, cstdint, dirent.h, etc.)

I would generally expect this:
```
#include "cli.hpp"

#include "conio.hpp"
#include "menuItems/systemCommands.hpp"
#include "states.hpp"

#include "Particle.h"

#include <fstream>
```

Reply:
Fixed here